src/producer/sqs.js

```diff
@@ -0,0 +1,82 @@
+// @flow
+
+import moment from 'moment';
+
+import sqs from '../config/sqs';
+import type { Configuration, Logger, Producer, IProducer, IRegistry, sqsUrls } from '../../types';
+
+class SqsProducer implements IProducer {
+  config: Configuration;
+  registry: IRegistry;
+  logger: Logger;
+  producer: Producer;
+  sqsUrls: sqsUrls;
+
+  constructor(config: Configuration, registry: IRegistry, logger: Logger) {
+    this.config = config;
+    this.producer = sqs(config);
+    this.logger = logger;
+    this.registry = registry;
+  }
+
+  initialize = (topic: ?string) => {
```

Comment:
I was mixing methods & constants in a few places. 👎  Fixing it in the new changes. Added it to check some binding issues.